src/js/typesense/TypesenseClient.ts

```diff
@@ -68,9 +68,8 @@ export async function multiSearch (query: string): Promise<MultisearchReturnType
     searches: [
       {
         q: query,
-        query_by: 'climbName, climbDesc',
+        query_by: 'climbName, areaNames',
         collection: 'climbs',
-        exclude_fields: 'climbDesc',
```

Comment:
We still want to exclude `climbDesc` so that it's not included in the result json (to keep the payload light)

<img width="1065" alt="Screenshot 2023-07-11 at 8 12 39 PM" src="https://github.com/OpenBeta/open-tacos/assets/3805254/406740a2-30d0-4849-ac1a-2375e4235c88">

Reply:
Ah, yes!

Reply:
fix that now